plugins/aea-cli-ipfs/setup.py

```diff
@@ -40,6 +40,8 @@
     install_requires=[
         "open-aea>=1.0.0, <2.0.0",
         "ipfshttpclient>=0.8.0a2",
+    ],
+    tests_require=[
```

Comment:
pytest is required for plugin test tools - https://github.com/valory-xyz/open-aea/blob/45f9c0d01ede35b1d1b33b4e4a6c49304fef49aa/plugins/aea-cli-ipfs/aea_cli_ipfs/test_tools/fixture_helpers.py#L24

Reply:
Oops, sorry, I didn't know that. I thought it was just for testing purposes, like in other cases.

Will it work with pytest 8? If `pytest` is a forced dependency, it's sad to force users to use pytest <7.3.0 in their projects.

Reply:
But in any case, I reverted to the original form 👌